CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## PaymentSheet
 * [Fixed] Fixed an issue that caused animations of the card logos in the Card input field to glitch.
 * [Fixed] Fixed a layout issue in the "Save my info" checkbox.
+* [Fixed] Fixed a bug where 3 decimal place currencies were not being formatted properly.
```

Comment:
Is this specific to PaymentSheet?

Reply:
Technically no, but that's where the bug was reported. Maybe we can just put it under a `Payments` header if that makes more sense.

Reply:
@yuki-stripe updated the change log to put it under "Payments"

Reply:
Looking at usage of this method, the bug seems to affect ~all UI surfaces that show an amount. I'm not sure what "Payments" refers to, I think we can just put at the top-level.